### DISCUSSION TITLE
Hearing Mod is glitching out the ship lever

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#338] [Q&A] [answered]
So if i enable /hear on start round, we collect some items and then fly back to the orbit, we will be stuck in a place where its not planet nor orbit. We cannot go to any moons and the game didn't save the file.
But if i do /hear to turn the Hearing Mod before we fly back to orbit, our ship will be fine.

What caused it?

**Comment by winstxnhdw** [comment, marked as answer]
I thought I fixed this. Are you on latest?

**Reply by Dexteriority** [reply]
i will redownload it

**Reply by Dexteriority** [reply]
Yep you did, i just didn't check the update, sorry.
